.github/workflows/build-fb-image.yaml

```diff
@@ -36,18 +45,100 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
 
       - name: Login to Docker Hub
         uses: docker/login-action@v3
         with:
+          registry: docker.io
           username: ${{ secrets.REGISTRY_USER }}
           password: ${{ secrets.REGISTRY_PASSWORD }}
 
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v3
-
+      
       - name: Build and Push Image for Fluent Bit
-        run: |
-          make build-fb -e FB_IMG=${{ env.FB_IMG }}
-          make build-fb-debug -e FB_IMG_DEBUG=${{ env.FB_IMG_DEBUG }}
\ No newline at end of file
+        id: docker-build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./cmd/fluent-watcher/fluentbit/Dockerfile
+          push: true
+          platforms: linux/amd64,linux/arm64
```

Comment:
This will be pretty slow btw, it may not be worthwhile to do ARM builds for PRs.

Reply:
We are currently building in our existing approach, so I have added.

https://github.com/fluent/fluent-operator/blob/master/Makefile#L99-L103